docs/generalqa.md

```diff
@@ -34,6 +34,39 @@ To contribute to this section, please review our [Code of Conduct](CODE_OF_CONDU
 
 ---
 
+## SMTP Configuration
+**Q:** How can I enable SMTP for Archipelago?
+
+**A:** For standard demo deployments, SMTP is not setup to send emails. To enable SMTP:
+
+1. Run the following commands in your terminal.
```

Comment:
I would just add a note so people are sure that docker is running. a docker ps? and check if all is there?

Reply:
Good idea, added!